### ISSUE TITLE
Section "Couverture géographique" des fiches de MD, informations renseignés par défauts

### ISSUE THREAD

**Issue by StormKlaus** [MTES-MCT/metadata-postgresql#103]
Dans la section "Couverture géographique" des fiches de MD j'ai des informations dans les champs "Rectangle d'emprise", "Centroïde", et "Géométrie"
alors que ma table en test est une donnée non géographique....

Ces informations ne sont pas retranscrites en Consultation car ce sont des exemples par défaut ? 

Je sais pas s'il est préférable de laisser ces informations par défaut ou de les ôter si non calculées...

Qu'en pensez-vous ?

**Comment by alhyss** [comment]
Oui, c'est un _placeholder_. Le texte est grisé et disparaît dès que tu cliques sur le champ. L'objectif était de montrer la syntaxe attendue, pour ceux qui aurait besoin de saisir la valeur manuellement parce que les outils de calcul automatique ou de tracé manuel ne répondent pas à leur besoin.

Je conçois que ça peut paraître perturbant, mais je ne suis pas trop favorable à l'idée d'aller conditionner l'apparition d'un `placeholder` à des caractéristiques de la table. C'est sans doute faisable mais ce sera long à implémenter, et je ne crois pas que le jeu en vaille la chandelle.

D'autant qu'un service qui voudrait s'en débarrasser (quelle que soit la table) a déjà une solution : activer PlumePg sur sa base, charger au moins les modèles par défaut, et supprimer le _placeholder_.

Pour l'ensemble des modèles : 
```sql
UPDATE z_plume.meta_categorie
    SET placeholder = ''
    WHERE path IN (
         'dct:spatial / dcat:bbox', 
         'dct:spatial / dcat:centroid', 
         'dct:spatial / locn:geometry'
    ) ;
```

Tu peux même faire du conditionnel si tu crées des modèles spécifiques pour tes tables non géométriques, avec une condition d'application automatique basée par exemple sur la présence d'un champ de géométrie, et que tu retires uniquement le _placeholder_ pour ces modèles. 

Pour un modèle donné : 
```sql
UPDATE z_plume.meta_template_categories
    SET placeholder = ''
    WHERE shrcat_path IN (
         'dct:spatial / dcat:bbox', 
         'dct:spatial / dcat:centroid', 
         'dct:spatial / locn:geometry'
    ) AND tpl_label = 'Mon modèle sans géométrie' ;
```

Mais tu ferais peut-être aussi bien de retirer carrément la métadonnée _Couverture géographique_ dans ce cas...

Évidemment, le _placeholder_ sera toujours là dans le cas où aucun modèle n'est utilisé, puisque c'est alors le paramétrage du schéma commun qui s'applique. Si tu le souhaites, on peut poser la question lors du prochain sous-groupe Métadonnées : est-ce qu'il leur semble souhaitable ou non de conserver ce _placeholder_ dans le schéma commun ?

**Comment by StormKlaus** [comment]
La conception d'un modèle local (ou pré-implémenté) pour les données non géographiques me semble bien être la solution,
afin d'épurer les éléments de MD géographiques.

Aussi, on pourrait même créer (localement), pour les données géographiques, un modèle par géométrie (pont, ligne, polygone) afin d’épurer les champs de géométries non raccords avec la donnée, au besoin.

Ainsi, on est libre de garder ou non ces éléments pouvant être perturbant.

Je pense, pour toutes les raisons que tu as donné, que l'on peut conserver ce placeholder dans le schéma commun.

Merci pour tous ces éléments.
